internal/lib/lib.go

```diff
@@ -267,38 +279,52 @@ func GetEvhVsPoolNPgName(ingName, namespace, host, path, infrasetting string, ar
 		svcName := args[0]
 		poolName = poolName + "-" + svcName
 	}
+	return Encode(poolName)
+}
+func GetEvhPoolName(ingName, namespace, host, path, infrasetting, svcName string) string {
```

Comment:
GetEvhPoolName and GetEvhVsPoolNPgName can potentially create a confusion. GetEvhVsPoolNPgName gives us the encoded name and GetEvhPoolName gives us the unencoded name. I think we came rename this function, e.g. GetEvhPoolNameUnencoded (or anything suitable) to indicate that this is not encoded pool name, else in future we may end up using this function for populating pools. 

Currently GetEvhPoolName and GetEvhVsPoolNPgName are independent, so one can make a logical change in GetEvhPoolName without changing GetEvhVsPoolNPgName and it would create problems. We can call GetEvhPoolName from GetEvhVsPoolNPgName and then apply encoding on the return value. 

Though not introduced by this PR, we can thin about breaking up GetEvhVsPoolNPgName - one for PG, one for Pool. I think initially same function was used for both but now we need sic name for pool, but it is not needed for PG, so we can use different functions.